app/src/androidTest/java/org/mozilla/focus/activity/AddtoHSTest.java

```diff
@@ -243,8 +243,7 @@ public void SearchTermShortcutTest() throws InterruptedException, UiObjectNotFou
         TestHelper.inlineAutocompleteEditText.setText("helloworld");
         TestHelper.hint.waitForExists(waitingTime);
         TestHelper.pressEnterKey();
-        TestHelper.progressBar.waitForExists(webPageLoadwaitingTime);
-        Assert.assertTrue(TestHelper.progressBar.waitUntilGone(webPageLoadwaitingTime));
+        TestHelper.progressBar.waitUntilGone(webPageLoadwaitingTime);
```

Comment:
Can you explain why you removed the assertion? Without the assert this is basically just a sleep, isn't it?

Reply:
I think .waitUntilGone gets awaken when the event (progressbar disappearing) gets triggered, so it's technically not a sleep().  At the time, I thought assert was failing when there is no progressBar to disappear.  (it was exiting right away)  I'll double check though because I can't find doc that confirms/denies what I saw.